### PR TITLE
Clean up Kotlin guidelines

### PR DESCRIPTION
**Pull request by sindrenm**

### CODE REVIEW

File: kotlin-code-style.md [hunk 1]
```diff
@@ -36,48 +40,49 @@ class Person(val firstName: String, val lastName: String, var age: Int) {
...
-#Object Expressions and Declarations
+# Object Expressions and Declarations
 
-##Object expressions
+## Object expressions
 
-To avoid too long method, we should externalize object expressions as properties
-:
+To avoid writing methods that are too long, externalize object expressions as
+properties:
```
Comment (jeantuffier) [inline kotlin-code-style.md:60]: maybe we should specifies something like "unless you can use a direct lambda expression or if the actual object expression is not so long, like ~5 lines max or something"?

File: kotlin-code-style.md [hunk 2]
```diff
@@ -36,48 +40,49 @@ class Person(val firstName: String, val lastName: String, var age: Int) {
...
+have lazy loading properties:
 
 ```kotlin
 val myValue : String by lazy {
-    someInit() //executed only at the first access
-    callToAFunctionReturningAStringValue()
+    someInit()
+    functionReturningAStringValue()
 }
 ```
```
Comment (jeantuffier) [inline kotlin-code-style.md:88]: well in the end `lazy` delegates are not so much used, at least by me ahah. And it really depends of the context, so I think this part can be removed from the play book

File: kotlin-code-style.md (hunk 1 above)
Comment (sindrenm) [inline kotlin-code-style.md:60]: Well, honestly I'm not entirely sure if I agree with this one, but I reworded it nevertheless. :smiley:

File: kotlin-code-style.md (hunk 2 above)
Comment (sindrenm) [inline kotlin-code-style.md:88]: I agree, but this is still only a rewording. But yeah, we shouldn't “use `lazy` as often as possible”, only when appropriate.

**Comment by jeantuffier** [comment]
merging this one. But I think we should change some stuff in here, like the things discussed in comments

**Comment by sindrenm** [comment]
Agreed. we already have https://github.com/hyperoslo/android-playbook/issues/24 addressing the lazy loading part.

**Pull request merged**